crates/holochain_metrics/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "holochain_metrics"
+version = "0.3.0-beta-dev.1"
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
+edition = "2021"
+description = "metrics helpers"
+license = "MIT"
+documentation = "https://docs.rs/holochain_metrics"
+repository = "https://github.com/holochain/holochain"
+
+[features]
+default = []
+
+[dependencies]
+influxive-child-svc = "=0.0.1-alpha.2"
+influxive-otel = "=0.0.1-alpha.2"
```

Comment:
could we start having these in-repo and push them out when they're stable?

Reply:
> could we start having these in-repo

No. For all the reasons I keep resisting attempts to put everything in our monorepo.

- It drastically slows down development time
- There's absolutely no reason to couple the release timing of these upstream dependencies with those of holochain, even though we're managing both
- Putting it in the monorepo conflates PRs and Issues that have no business being conflated
- If we're serious about trying to promote our own code as useful to the community at large, it'll be far easier to promote them in digestible chunks, rather than, "Hey, come look at our giant monorepo and try to comprehend it all at once".

Reply:
> push them out when they're stable?

They are pushed out. If you mean without the pre-release label, sure, we can publish them without that before merging this PR into Holochain.